src/lib/ClaimCodeManager.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import crypto from 'crypto';
+
+type ClaimCodeObject = {
+  claimCode: string,
+  userID: string,
+}
+
+const CLAIM_CODE_LENGTH = 63;
+const CLAIM_CODE_LIVE_TIME = 5000 * 60; // 5 min
+
+class ClaimCodeManager {
+  _claimCodes: Array<ClaimCodeObject> = [];
```

Comment:
I would definitely prefer using a `Map<string, string>` {claimCode, userID}`

I don't see a reason here why that wouldn't work.